kube/services/jobs/gentestdata-job.yaml

```diff
@@ -0,0 +1,139 @@
+# run with:
+# g3k runjob gentestdata \
+# SUBMISSION_HOST submission_host \
+# DICTIONARY_REPO dictionary_repo \
+# BRANCH  branch \
+# TEST_PROJECT "" \
+# NUMBER_OF_EXAMPLES "" \
+# NOTE: "" indicates an optional argument. Leave as "" for default
+#
+# SUBMISSION_HOST(required)
+#   url of submission host. e.g. https://giangb.planx-pla.net
+#
+# DICTIONARY_REPO(required)
+#   url of dictionary repo. e.g. https://github.com/uc-cdis/datadictionary
+#
+# BRANCH(optional)
+#   branch of the dictionary repos. Default: master
+#
+# TEST_PROJECT(optional)
+#   Name of test project. Default: DEV/test
+#
+# NUMBER_OF_EXAMPLES(optional)
+#   Number of examples for each node. Default 1000
+#
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: gentestdata
+spec:
+  # not yet supported - backOffLimit: 3
+  template:
+    spec:
+      serviceAccountName: useryaml-job
+      volumes:
+        - name: shared-data
+          emptyDir: {}
+        - name: config-volume
+          secret:
+            secretName: "fence-secret"
+        - name: creds-volume
+          secret:
+            secretName: "fence-creds"
+        - name: config-helper
+          configMap:
+            name: config-helper
+        - name: json-secret-volume
+          secret:
+            secretName: "fence-json-secret"
+        - name: fence-jwt-keys
+          secret:
+            secretName: "fence-jwt-keys"
+      containers:
+      - name: fence
+        image: quay.io/cdis/auto-qa:feat_gen_test_data
+        imagePullPolicy: Always
+        env:
+          - name: PYTHONPATH
+            value: /var/www/fence
+          - name: DICTIONARY_REPO
```

Comment:
Hey Giang,

The DICTIONARY_REPO should come from the global configmap, because that's the dictionary the gdcapi database is configured for.  Sorry for the confusion ...

Reply:
It works on genomel now. Francisco is fixing the dictionary schemas.

Reply:
fixed at 869720dd591a484a389b1cc565eb91dfc4c99f48